runtime/gc_realtime/RealtimeMarkingSchemeRootClearer.hpp

```diff
@@ -42,7 +42,7 @@
  */
 class MM_RealtimeMarkingSchemeRootClearer : public MM_RealtimeRootScanner
 {
-/* Data memebers / types */
+/* Data members / types */
```

Comment:
this change used to be in runtime/gc_realtime/RealtimeMarkingScheme.cpp before the conflict